lib/clientSideScripts/hideScrollbars.ts

```diff
@@ -2,9 +2,13 @@
  * Hide the scrollbars
  */
 export default function hideScrollBars(hide: boolean): void {
-  if (hide) {
-    document.body.style.overflow = 'hidden';
-  } else {
-    document.body.style.overflow = '';
+  try {
+    if (hide) {
+      document.body.style.overflow = 'hidden';
+    } else {
+      document.body.style.overflow = '';
+    }
+  } catch (e) {
+    // Do nothing
```

Comment:
Is it worth logging something here?

Reply:
This is logged into browser where the test is executed, it doesn't return anything